storage/file/file.go

```diff
@@ -0,0 +1,29 @@
+package file
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+)
+
+// Provider implements a file-based image storage
+type Provider struct {
+	path string
+}
+
+// New returns a new Provider instance
+func New(path string) (*Provider, error) {
```

Comment:
If the path is user settable you should ensure the path is writable and have error handling here.

Reply:
👍